collector/os_release.go

```diff
@@ -128,34 +127,23 @@ func parseOSRelease(r io.Reader) (*osRelease, error) {
 }
 
 func (c *osReleaseCollector) UpdateStruct(path string) error {
-	releaseFile, err := os.Open(path)
+	path, err := filepath.EvalSymlinks(path)
```

Comment:
We shouldn't need to do the symlink bit if we're not checking the mtime anymore.